content/community.md

```diff
@@ -1,29 +1,41 @@
-# Welcome to Locally Optimistic
+---
+title: "Getting Involved"
+date: "2018-06-10"
+layout: "about"
+menu: "main"
+comments: false
+weight: 50
+---
 
-We are thrilled you have decided to join us.
+## Joining the Community
 
-If you have not already, please [reach out](mailto:kaminsky.michael@gmail.com) to ask for an invite to our slack group. 
+We are thrilled you have decided to join us. We have a growing [slack community](locallyoptimistic.slack.com) where current and aspiring analytics leaders discuss and share lessons and challenges from their experience working with data.
```

Comment:
I think you may need to add `https:://www.` for the slack link. The link didnt work for me on local

Reply:
this is required but there is only one `:` in your standard `http://` situation :)